src/components/navigation/header/Header.tsx

```diff
@@ -85,6 +86,9 @@ export const Header = ({ data, assets }: IHeader) => {
             )}
             {renderPageLinks(links, false, pathname)}
             {renderPageCTAs(ctas, false)}
+            <div className={styles.languageSwitcher}>
```

Comment:
remember to make sure it's visible in the mobile meny as well

Reply:
Good point, fixed now ✅